src/main/java/com/danichef/rest/player/seats/PlayerSit.java

```diff
@@ -1,9 +1,26 @@
 package com.danichef.rest.player.seats;
 
+import org.bukkit.entity.Entity;
 import org.bukkit.entity.Player;
 
 public interface PlayerSit {
+
+    /**
+     * Makes player sit
+     * @param player to sit
+     */
     void sit(Player player);
 
+    /**
+     * Makes player stand up
+     * @param player to stand up
+     */
     void standUp(Player player);
+
+    /**
+     * Returns whether the given entity is a seat or not
+     * @param entity to check
+     * @return whether the given entity is a seat or not
+     */
+    boolean isSeat(Entity entity);
```

Comment:
As this method will most likely be called by other plugins, inserting the jetbrains annotation [@NotNull](https://www.jetbrains.com/help/idea/nullable-and-notnull-annotations.html) might be a good idea to ensure a non-null entity being passed.